### DISCUSSION TITLE
Changing the Campaign Name

### DISCUSSION THREAD

**Discussion by knyghtmare** [nemaara/Northern_Rebirth_Reborn#4] [Ideas]
Given how this campaign has scenario names, structure and a plot which is drastically different from either NR or THoT, I think the campaign name should be changed.
This might resolve some emergent problems:

1. "NR walkthrough" misleading players. Context: we have the usual confused player asking "Where is Ratheln?" since the campaign was revised and walkthrough staying the same (not updated).
2. The returnee confusion. Players dont stay constant and usually take monthly/yearly gaps before re-installing. When they do, the confusion arises. "Wait, I dont remember this" / "Where's the Camerin?" / "Krash?". A new campaign name would help resolve such issues as they would check it and see the changelog which would clearly state that it's the merger of NR and THoT. There is also a similar case seen for returning players experiencing the reworked DiD. "Wait, what the heck is the third scenario? I dont remember this at all." Amusing reads apparently.
3. Helps resolve the question: "If you had to rework NR, why couldn't you just give the dark branch back?" They won't be asking it if the campaign has a new name.

**Comment by cooljeanius** [comment]
Yeah agreed that renaming would be good if the changes are going to be that drastic; also the acronym "NRR" conflicts with the other currently-existing NR fork, "Northern Rebirth Remake", at least how I modded it: https://github.com/cooljeanius/wesnoth_mods/blob/master/campaigns/Northern_Rebirth_Remake/diffs/_main.diff

**Reply by knyghtmare** [reply]
I am still a bit perplexed with how this Discussions / Q-A of this Github works. So far, it seems like a mix of forums and Quora. Anyways, yeah, time to come up with a list of names, I suppose.

**Comment by knyghtmare** [comment]
May we have been looking at this the wrong way...
Like re-reading the draft I made, and seeing who the primary antagonist is, I think this isn't NR absorbing THoT, rather the opposite as in THoT incorporating NR into it, as the plot being followed is more related to THoT. Thus, maybe just rename to (The) Hammer of Thursagan? Or, maybe Struggle for the Northlands (SftN) or...For the Northlands (FtN)?
